armadillo/src/main/java/org/molgenis/armadillo/metadata/UserDetails.java

```diff
@@ -53,4 +53,8 @@ public static UserDetails create(
   public static UserDetails create(String newEmail) {
     return new AutoValue_UserDetails(newEmail, null, null, null, null, new HashSet<>());
   }
+
```

Comment:
Duplicate?

Reply:
No it's specifically for easily creating an admin. `create` vs `createAdmin`

Reply:
ah, I should read better, thx.